packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -70,33 +73,116 @@ function TagGroup<T extends object>(props: SpectrumTagGroupProps<T>, ref: DOMRef
   }, state, domRef);
   const {tagGroupProps} = useTagGroup(props);
 
+  let [isCollapsed, setIsCollapsed] = useState(maxRows != null);
+  let [tagState, setTagState] = useValueEffect({visibleTagCount: gridCollection.size, showCollapseButton: false});
+
+  let updateVisibleTagCount = useCallback(() => {
+    if (maxRows > 0) {
+      let computeVisibleTagCount = () => {
+        // Refs can be null at runtime.
+        let currDomRef: HTMLDivElement | null = domRef.current;
+        if (!currDomRef) {
+          return;
+        }
+
+        let [tags, button] = [...currDomRef.children].reduce((acc, el) => {
```

Comment:
What is acc short for?

Reply:
it's standard shorthand for accumulator in a reduce function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#grouping_objects_by_a_property